index.js

```diff
@@ -56,34 +57,87 @@ function HyperString (db, opts) {
   })
 }
 
-HyperString.prototype.insert = function (prev, chr, cb) {
-  // TODO: check that it's a single character
-  // TODO: support inserting a whole string, and breaking it up into
-  // one-char inserts
-  var op = {
-    op: 'insert',
-    chr: chr,
-    prev: prev || null
+HyperString.prototype.insert = function (prev, string, done) {
```

Comment:
Yes, I was thinking about this. If hyperlog could help us that would be awesome.